test/e2e/support/utils.js

```diff
@@ -107,6 +107,6 @@ export async function pressWithModifier( modifier, key ) {
  *
  * @return {Promise} Promise resolving after the given timeout
  */
-export async function wait( timeout ) {
+export async function waitDurationYesImCertainWaitForSelectorIsNotABetterOption( timeout ) {
```

Comment:
I wonder if we could just name it `waitForAnimation` (that's the current use-case) Or maybe just leave it as `wait` and add a comment.